scalableminds/webknossos
# Enforce Pricing for Team Sharing

PR adds a new wrapper component blur components/features that are not part of an organization's selected plan.

Changes:
- new blurring wrapper component
- affects Team sharing modal and dataset setting's sharing tab
- moved `PricingPlanEnum` to `pricing_plan_utils` to unroll a cyclic dependency 

### URL of deployed dev instance (used for testing):
- https://moreplans.webknossos.xyz/

### Steps to test:
- Set orga plan accordingly in your DB
- Check annotation team sharing modal
- Check dataset sharing tab

### Issues:
- contributes to #6509 

------
(Please delete unneeded items, merge only when none are left open)
